settings/acsf/post-install/post-install.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Factory Hook: post-install.
+ *
+ * This hook enables you to execute PHP after a new website is created
+ * in your subscription. Unlike most API-based hooks, this hook does not
+ * take arguments, but instead executes the PHP code it is provided.
+ */
+
+$site = $_ENV['AH_SITE_GROUP'];
+$env = $_ENV['AH_SITE_ENVIRONMENT'];
+
+// The public domain name of the website.
+// Run updates against requested domain rather than acsf primary domain.
+$domain = $_SERVER['HTTP_HOST'];
+
+
+exec('/mnt/www/html/$target_env/vendor/acquia/blt/bin/blt deploy:update --define environment=$env --define drush.uri=$domain -v -y');
```

Comment:
I don't think that variable substitution will work here since you're using single quotes.

Reply:
Please use the long form of all options. E.g., `--verbose --yes`.